style.css

```diff
@@ -0,0 +1,817 @@
+  padding: 0 35px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu-btn,
+.nav-menu > button {
+  background-color: white;
+  margin-right: 40px;
+}
+
+.nav-menu > button {
+  margin: 0.5em;
+  align-self: flex-end;
+  cursor: pointer;
+}
+
+.nav-menu > img {
+  width: 2em;
+}
+
+.nav-menu > ul > li {
+  margin-left: 28px;
+  margin-top: 40px;
+  font-size: 24px;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+.nav-menu > ul > li > a {
+  font-size: 24px;
+  font-weight: 600;
+  color: #3a4a42;
+  cursor: pointer;
+}
+
+.link-portfolio:hover,
+.link-about:hover,
+.link-contact:hover {
+  color: var(--color-primary);
+}
+
+.btn-close-nav {
+  position: absolute;
+  font-size: 30px;
+  right: 48px;
+  top: 20px;
+}
+
+.nav-menu {
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  top: 0;
+  right: -100%;
+  background-color: #fff;
+  backdrop-filter: blur(8px);
+  overflow-x: hidden;
+  overflow-y: hidden;
+  display: flex;
+  flex-direction: column;
+  transition: 0.5s;
+}
+
+.show {
+  left: 0;
+}
+
+/* .menu-btn.hidden {
+  display: none;
+} */
+
+/* .nav-menu.hidden {
+  display: none;
+} */
+
+.header-logo {
+  text-decoration: none;
+  font-family: inter, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.headline {
+  background: #fff url("./images/background.png") no-repeat;
+  background-size: cover;
+  overflow: hidden;
+  background-position: center;
+}
+
+/* .headline h1 {
+  margin-top: 150px;
+} */
+
+.headline p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.headline button {
+  background: none;
+}
+
+.social-media {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  width: 170px;
+}
+
+@media (min-width: 992px) {
+  .menu-btn {
+    display: none;
+  }
+
+  .header-logo {
+    display: none;
+  }
+
+  .header-title {
+    padding-top: 3em;
+  }
+
+  .header-nav-desktop {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 0;
+    padding-top: 1em;
+    margin-left: 500px;
+  }
+
+  .header-nav-desktop li {
+    list-style: none;
+    padding: 0;
+    font-size: 18px;
+    line-height: 42px;
+    font-weight: bold;
+    color: #42526e;
+    cursor: pointer;
+  }
+
+  .headline {
+    background: #fff url("./images/background.png") no-repeat;
+    background-size: 100%;
+    padding-bottom: 8em;
+  }
+}
+
+@media (min-width: 1024px) {
+  .headline {
+    padding-bottom: 15em;
+  }
+
+  .about-myself {
+    background:
+      url("./images/frame5.png") no-repeat 0% 20%,
+      url("./images/frame4.png") no-repeat 100% 0%;
+  }
+}
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(3, auto);
+  padding: 1.5rem;
+  text-align: center;
+  column-gap: 1rem;
+  width: 90%;
+  margin: 0 auto;
+}
```

Comment:
- Hey, Please, let's kindly adjust our styles as your card are having too much space between each other compared to the Figma design. see the images below:

*Figma design*
![Screenshot from 2023-01-19 18-14-35](https://user-images.githubusercontent.com/59397159/213514905-b85b5828-d9e7-4b86-b86e-8e84d6dccd80.png)


*your design*
![Screenshot from 2023-01-19 18-13-50](https://user-images.githubusercontent.com/59397159/213514914-b923f56f-2cd1-4bc3-8c0c-616c38d0ea99.png)